### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by adilsoybali**
# Description

Installing executables with "go get" in module mode is deprecated.
"go install pkg@version" should be used instead.
For more information, see https://golang.org/doc/go-get-install-deprecation

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -35,3 +35,4 @@
 * [SolomonSklash](https://github.com/SolomonSklash)
 * [l4yton](https://github.com/l4yton)
 * [xfgusta](https://github.com/xfgusta)
+* [adilsoybali](https://github.com/adilsoybali)
```
Comment (joohoi) [inline CONTRIBUTORS.md:38]: This list should be kept in alphabetically sorted order, can you make that change?

**Review by joohoi — CHANGES_REQUESTED** [review]
Thanks for the PR! This is a good documentation change. 

However, I think the required Go version mentioned [here](https://github.com/ffuf/ffuf/blob/master/README.md?plain=1#L50) should be changed to 1.16 at the same time in order to not to cause confusion if `go install` doesn't work as expected.

**Comment by adilsoybali** [comment]
Hi @joohoi. I made the changes you requested. If there's anything else that needs to change, I can do that too. Thank you, have a nice day.

**Review by joohoi — APPROVED** [review]
Thanks for the quick response. It's all good now, merging!

**Pull request merged**